src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuefinaldecision/IssueFinalDecisionSubmittedHandlerTest.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.issuefinaldecision;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+import static org.mockito.MockitoAnnotations.openMocks;
+import static uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType.SUBMITTED;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.NO;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.YES;
+
+import java.util.Optional;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import junitparams.JUnitParamsRunner;
+import org.hibernate.validator.messageinterpolation.ParameterMessageInterpolator;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.ccd.service.CcdCallbackMapService;
+import uk.gov.hmcts.reform.sscs.service.*;
+
+@RunWith(JUnitParamsRunner.class)
+public class IssueFinalDecisionSubmittedHandlerTest {
+    private static final String USER_AUTHORISATION = "Bearer token";
+    private IssueFinalDecisionSubmittedHandler handler;
+
+    @Mock
+    private Callback<SscsCaseData> callback;
+    @Mock
+    private CaseDetails<SscsCaseData> caseDetails;
+    @Mock
+    private CcdCallbackMapService ccdCallbackMapService;
+    @Mock
+    private CallbackOrchestratorService callbackOrchestratorService;
+
+    private SscsCaseData sscsCaseData;
+
+    protected static Validator validator = Validation.byDefaultProvider()
+            .configure()
+            .messageInterpolator(new ParameterMessageInterpolator())
+            .buildValidatorFactory()
+            .getValidator();
+
+    @Before
+    public void setUp() {
+        openMocks(this);
```

Comment:
Remove this and use ExtendWith(MockitoExtension.class) annotation on class instead

Reply:
cant remove this as it complains